diff --git a/server/src/main/java/de/thkoeln/cexprotobuf/server/CoffeeOrderJsonController.java b/server/src/main/java/de/thkoeln/cexprotobuf/server/CoffeeOrderJsonController.java
index b8824c7..5696545 100644
--- a/server/src/main/java/de/thkoeln/cexprotobuf/server/CoffeeOrderJsonController.java
+++ b/server/src/main/java/de/thkoeln/cexprotobuf/server/CoffeeOrderJsonController.java
@@ -23,7 +23,7 @@ public class CoffeeOrderJsonController {
   static {
     var order = new OrderJsonDto(
         "Cappuccino",
-        CupSizeJsonDto.TALL,
+        CupSizeJsonDto.VENTI,
         Instant.now(),
         2,
         List.of(
diff --git a/server/src/main/java/de/thkoeln/cexprotobuf/server/messages/json/CupSizeJsonDto.java b/server/src/main/java/de/thkoeln/cexprotobuf/server/messages/json/CupSizeJsonDto.java
index a769204..902af0c 100644
--- a/server/src/main/java/de/thkoeln/cexprotobuf/server/messages/json/CupSizeJsonDto.java
+++ b/server/src/main/java/de/thkoeln/cexprotobuf/server/messages/json/CupSizeJsonDto.java
@@ -1,5 +1,5 @@
 package de.thkoeln.cexprotobuf.server.messages.json;
 
 public enum CupSizeJsonDto {
- SMALL, TALL, GRANDE, VENTI;
+ SMALL, GRANDE, VENTI;
 }
