diff --git a/server/src/main/java/de/thkoeln/cexprotobuf/server/CoffeeOrderProtobufController.java b/server/src/main/java/de/thkoeln/cexprotobuf/server/CoffeeOrderProtobufController.java
index 6f05bba..f386829 100644
--- a/server/src/main/java/de/thkoeln/cexprotobuf/server/CoffeeOrderProtobufController.java
+++ b/server/src/main/java/de/thkoeln/cexprotobuf/server/CoffeeOrderProtobufController.java
@@ -9,6 +9,7 @@ import de.thkoeln.cexprotobuf.messages.proto.OrderProtoDto;
 import java.time.Instant;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Random;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
@@ -44,7 +45,7 @@ public class CoffeeOrderProtobufController {
             .setPrice(20)
             .build())
         .setCash(CashProtoDto.newBuilder()
-            .setAmount(100)
+            .setAmount(new Random().nextDouble())
             .build())
         .setPrice(100)
         .setId(1)
diff --git a/server/src/main/proto/coffee_order.proto b/server/src/main/proto/coffee_order.proto
index a85df41..d628a26 100644
--- a/server/src/main/proto/coffee_order.proto
+++ b/server/src/main/proto/coffee_order.proto
@@ -31,7 +31,7 @@ message OptionProtoDto {
 }
 
 message CashProtoDto {
-  float amount = 1;
+  double amount = 1;
 }
 
 message CreditCardProtoDto {
