diff --git a/server/src/main/java/de/thkoeln/cexprotobuf/server/CoffeeOrderProtobufController.java b/server/src/main/java/de/thkoeln/cexprotobuf/server/CoffeeOrderProtobufController.java
index 6f05bba..ae4d890 100644
--- a/server/src/main/java/de/thkoeln/cexprotobuf/server/CoffeeOrderProtobufController.java
+++ b/server/src/main/java/de/thkoeln/cexprotobuf/server/CoffeeOrderProtobufController.java
@@ -25,7 +25,7 @@ public class CoffeeOrderProtobufController {
     var now = Instant.now();
     var order = OrderProtoDto.newBuilder()
         .setProduct("Cappuccino")
-        .setSize(CupSizeProtoDto.TALL)
+        .setSize("maxi")
         .setDate(Timestamp.newBuilder()
             .setSeconds(now.getEpochSecond())
             .setNanos(now.getNano())
diff --git a/server/src/main/proto/coffee_order.proto b/server/src/main/proto/coffee_order.proto
index a85df41..586e362 100644
--- a/server/src/main/proto/coffee_order.proto
+++ b/server/src/main/proto/coffee_order.proto
@@ -11,7 +11,7 @@ message OrderCollectionWrapperProtoDto {
 }
 
 message OrderProtoDto {
-  CupSizeProtoDto size = 1;
+  string size = 1;
   string product = 2;
   google.protobuf.Timestamp date = 3;
   int32 count = 4;
